lib/summarize.js

```diff
@@ -8,12 +8,11 @@ const pify = require('pify');
 
 function uglify(content) {
   return new Promise((resolve, reject) => {
```

Comment:
I believe the following is a more concise (with better stack-traces/debugging approach)

```js
async function uglify(content) {
  const { code } = await Tester.minify(content);
  return code;
}
```